speaker/bgp_controller_test.go

```diff
@@ -207,1034 +206,6 @@ func (s *testK8S) Errorf(_ *v1.Service, evtType string, msg string, args ...inte
-						{
-							Addresses: []v1.EndpointAddress{
-								{
-									IP:       "2.3.4.5",
-									NodeName: pointer.StrPtr("iris"),
-								},
-							},
-							NotReadyAddresses: []v1.EndpointAddress{
-								{
-									IP:       "2.3.4.6",
-									NodeName: pointer.StrPtr("pandora"),
-								},
-							},
-						},
-					},
-				},
-				Type: epslices.Eps,
-			},
-			wantAds: map[string][]*bgp.Advertisement{
-				"1.2.3.4:0": {
-					{
-						Prefix: ipnet("10.20.30.1/32"),
-					},
-				},
-			},
-			expectedCfgRet: controllers.SyncStateReprocessAll,
-			expectedLBRet:  controllers.SyncStateSuccess,
-		},
-
-		{
-			desc: "Multiple advertisement config",
-			config: &config.Config{
-				Peers: map[string]*config.Peer{
-					"peer1": {
-						Addr:          net.ParseIP("1.2.3.4"),
-						NodeSelectors: []labels.Selector{labels.Everything()},
-					},
-				},
-				Pools: &config.Pools{ByName: map[string]*config.Pool{
-					"default": {
-						CIDR: []*net.IPNet{ipnet("10.20.30.0/24")},
-						BGPAdvertisements: []*config.BGPAdvertisement{
-							{
-								AggregationLength: 32,
-								LocalPref:         100,
-								Communities: func() map[community.BGPCommunity]bool {
-									community1, _ := community.New("0:1234")
-									community2, _ := community.New("0:2345")
-									community3, _ := community.New("large:123:456:789")
-									return map[community.BGPCommunity]bool{community1: true, community2: true,
-										community3: true}
-								}(),
-								Nodes: map[string]bool{"pandora": true},
-							},
-							{
-								AggregationLength: 24,
-								LocalPref:         1000,
-								Nodes:             map[string]bool{"pandora": true},
-							},
-						},
-					},
-				}},
-			},
-			balancer: "test1",
-			svc: &v1.Service{
-				Spec: v1.ServiceSpec{
-					Type:                  "LoadBalancer",
-					ExternalTrafficPolicy: "Cluster",
-				},
-				Status: statusAssigned("10.20.30.1"),
-			},
-			eps: epslices.EpsOrSlices{
-				EpVal: &v1.Endpoints{
-					Subsets: []v1.EndpointSubset{
-						{
-							Addresses: []v1.EndpointAddress{
-								{
-									IP:       "2.3.4.5",
-									NodeName: pointer.StrPtr("iris"),
-								},
-							},
-						},
-					},
-				},
-				Type: epslices.Eps,
-			},
-			wantAds: map[string][]*bgp.Advertisement{
-				"1.2.3.4:0": {
-					{
-						Prefix:    ipnet("10.20.30.1/32"),
-						LocalPref: 100,
-						Communities: func() []community.BGPCommunity {
-							community1, _ := community.New("0:1234")
-							community2, _ := community.New("0:2345")
-							community3, _ := community.New("large:123:456:789")
-							return []community.BGPCommunity{community3, community1, community2}
-						}(),
-					},
-					{
-						Prefix:    ipnet("10.20.30.0/24"),
-						LocalPref: 1000,
-					},
-				},
-			},
-			expectedCfgRet: controllers.SyncStateReprocessAll,
-			expectedLBRet:  controllers.SyncStateSuccess,
-		},
-		{
-			desc: "Multiple advertisement config, one only for my node",
```

Comment:
nit: this and the next one do not have an epslice one with the same name, just making sure we're good with zapping them